lib/helpers/normalizeArgs.js

```diff
@@ -22,4 +30,22 @@ function normalizeArgs(registry, args) {
   return map(flattenArgs, getFunction);
 }
 
+function similarTasks(registry, queryTask) {
+  if (typeof queryTask !== 'string') {
+    return [];
+  }
+
+  var tasks = registry.tasks();
+  var similarTasks = [];
+  for (var task in tasks) {
+    if (tasks.hasOwnProperty(task)) {
+      var distance = levenshtein.get(task, queryTask);
+      if (distance < 3) {
```

Comment:
3 might be too much?

- `1 clean ->  clear`
- `2 clean -> cleansn`
- `3 build -> biuldk`

Reply:
Typescript uses `<3`, but often has a namespace that includes over 1300 globals. `<4` would probably work for smaller namespaces too.

Reply:
Sorry, that was an oversimplification. The limit is actually `Math.floor(0.4 * task.length) + 1`, so that you're allowed one mistake per 5 characters.